{{cookiecutter.project_name}}/databricks-config/prod/outputs.tf

```diff
@@ -5,31 +5,31 @@
 
 # Batch inference job
 
-output "{{cookiecutter.project_name}}_batch_inference_job_id" {
+output "prod_{{cookiecutter.project_name}}_batch_inference_job_id" {
```

Comment:
Not fully sure if we want to add these env prefixes since they're put into `staging.json` and `prod.json` respectively, but it could help make it clearer. Thoughts?

Reply:
If you were to change it, the mlflow recipe yaml profile for prod/staging also needs to be changed. They read the value from the terraform output files.
The model validation also read experiment name from the terraform outputs(but I will change the logic to directly read it from terraform, in this sprint).

I feel it not necessary to add hardcoded prefix. Same names with different location helped with code reuse.

Reply:
Yeah there's also a couple places in the GitHub and ADO workflows that would need to be changed as well but before going through the effort to change all that, wanted validation that this is the right approach

Reply:
Agree with @mingyu89 , we probably don't need to do it, I think it's fine to rely on the prod/staging directories to disambiguate.

Reply:
Great I'll revert this part then.